WebRoot/js/zimbraMail/calendar/view/ZmCalColView.js

```diff
@@ -689,16 +689,18 @@ ZmCalColView.prototype._createItemHtml = function(appt) {
 	var location = appt.getLocation();
 	location = location && location.length && !is60 ? "<div class='appt_location'>" + AjxStringUtil.htmlEncode(appt.getLocation()) + "</div>" : null;
 
-	if ((is30 || isAllDay) && this.view !== ZmId.VIEW_CAL_DAY) {
-        // fit as much of appt name as we can in one row, use ... if we have to truncate
+    if (is30 || isAllDay) {
```

Comment:
minor: indentation is bit off in entire function

Reply:
@silentsakky I fixed code indentation in the function "ZmCalColView.prototype._createItemHtml".